src/settings.js

```diff
@@ -1606,6 +1606,20 @@ var WASM2C = 0;
 // in a custom location.
 var SEPARATE_DWARF_URL = '';
 
+// Whether the program should abort when an unhandled WASM exception is encountered.
+// This makes the Emscripten program behave more like a native program where the OS
+// would terminate the process and no further code can be executed when an unhanled
```

Comment:
```suggestion
// would terminate the process and no further code can be executed when an unhandled
```

Reply:
I can't believe I how many times I've misspelled this... ಠ_ಠ